src/app.js

```diff
@@ -184,56 +177,57 @@ var run = function () {
   var editor = new Editor(document.getElementById('input'))
 
   // ---------------- FilePanel --------------------
+  // TODO: All FilePanel related CSS should move into file-panel.js
+  // app.js provides file-panel.js with a css selector or a DOM element
+  // and file-panel.js adds its elements (including css), see "Editor" above
```

Comment:
The `.filepanel` class below should actually be named `.filepanel-container` or something.
It get's assigned to the `filepanelContainer` in line `190` below.

So basically, the css here is not related to the "file-panel.js" or anything, but it's more related to the code that uses the file-panel and wants to position it or put it in the place it wants to use it. So i don't think it belongs into the "filepanel" itself.

In theory, a different codebase could re-use the file panel and append it to a container that puts it in a different place, so it would use different css than the one below.